sources/packages/web/src/views/institution/locations/confirmation-of-enrollment/COESummaryData.vue

```diff
@@ -1,75 +1,98 @@
 <template>
-  <v-row class="m-2">
-    <v-col class="category-header-medium color-blue">{{ header }}</v-col>
-  </v-row>
-  <div class="mx-5 py-4">
-    <content-group>
-      <DataTable
-        :autoLayout="true"
-        :value="applications"
-        class="p-m-4"
-        :paginator="true"
-        :rows="10"
+  <body-header
+    :title="header"
+    :subTitle="subTitle"
+    :recordsCount="disbursements.coeSummary?.length"
+    class="m-1"
+  >
+    <template #actions>
+      <InputText
+        type="text"
+        placeholder="Search name"
+        v-model="searchCriteria"
+        @keyup.enter="searchCOE"
+      />
+      <v-btn class="ml-2 primary-btn-background" @click="searchCOE"
+        ><font-awesome-icon :icon="['fas', 'search']" class="mr-2"
+      /></v-btn>
+    </template>
+  </body-header>
+  <content-group>
+    <DataTable
+      :value="disbursements.coeSummary"
+      :lazy="true"
+      class="p-m-4"
+      :paginator="true"
+      :rows="pageLimit"
+      :rowsPerPageOptions="rowsPerPageOptions"
+      :totalRecords="disbursements.totalRecords"
+      @page="pageEvent"
+      @sort="sortEvent"
+    >
+      <template #empty>
+        <p class="text-center font-weight-bold">No records found.</p>
+      </template>
+      <Column field="fullName" header="Name" sortable="true">
```

Comment:
since u have already defined the datatable enum, 
![image](https://user-images.githubusercontent.com/77353155/153685034-79c36719-7018-4d34-927b-82fdd2100225.png), you can pass the field from the enum like `:field= "COESummaryFields.FullName"